Sources/ArcGISToolkit/Components/Offline/OnDemand/OnDemandListItemView.swift

```diff
@@ -32,6 +32,9 @@ struct OnDemandListItemView: View {
         selectedMap?.item?.title == model.title
     }
     
+    /// A Boolean value indicating whether the view should dismiss.
+    var shouldDismiss = true
```

Comment:
It would be better to not have multiple places where the default value exists, thus requiring passing it to the constructor of this view. Same for the other view.

```suggestion
    /// A Boolean value indicating whether the view should dismiss.
    let shouldDismiss: Bool
```